### PR TITLE
Tweaks for Dry-Run in Test Environment

### PR DESCRIPTION
**Pull request by mmarcotte**
This work includes the following:

- Following other patterns in the code base, if a petitioner record exists in Cognito, attempt to create the record in DynamoDB anyhow. We are doing this because we are creating the Cognito records ahead of time. This function currently catches instances where this isn't the case, and then attempts to create that record.
- Disable Smoketests for `test` and `migration` environment.
- Suppress welcome email when creating Petitioner record during Migration process
- Adjust script to bulk import practitioners
- Ensures proper env variables are configured when performing the switch-colors step locally

### CODE REVIEW

**Review by adunkman — APPROVED** [review]
Looks good!

File: .circleci/config.yml [hunk 1]
```diff
@@ -951,9 +951,7 @@ workflows:
             branches:
               only:
                 - irs
-                - migration
```
Comment (mmarcotte) [inline .circleci/config.yml:954]: don't want to perform smoketests in `migration` or `test` environment that would contaminate migration data

File: shared/src/persistence/dynamo/users/createMigratedPetitionerUser.js [hunk 2]
```diff
@@ -36,6 +36,7 @@ exports.createMigratedPetitionerUser = async ({ applicationContext, user }) => {
     const response = await applicationContext
       .getCognito()
       .adminCreateUser({
+        MessageAction: 'SUPPRESS',
```
Comment (mmarcotte) [inline shared/src/persistence/dynamo/users/createMigratedPetitionerUser.js:39]: don't want to send emails during migration for petitioners who haven't already been created

File: shared/src/persistence/dynamo/users/createMigratedPetitionerUser.js [hunk 3]
```diff
@@ -65,6 +66,34 @@ exports.createMigratedPetitionerUser = async ({ applicationContext, user }) => {
       userId: response.User.Username,
     });
   } catch (err) {
+    // the user already exists in Cognito
     applicationContext.logger.error(err);
+    const response = await applicationContext
```
Comment (mmarcotte) [inline shared/src/persistence/dynamo/users/createMigratedPetitionerUser.js:71]: reusing existing pattern for trying to get the user from Cognito if creating the user in Cognito fails. Then proceed with creating the DynamoDB record...

File: shared/src/persistence/dynamo/users/createPractitionerUser.js [hunk 4]
```diff
@@ -52,8 +52,13 @@ exports.createUserRecords = async ({ applicationContext, user, userId }) => {
 
 exports.createPractitionerUser = async ({ applicationContext, user }) => {
   let userId = applicationContext.getUniqueId();
+  const practitionerRoleTypes = [
```
Comment (mmarcotte) [inline shared/src/persistence/dynamo/users/createPractitionerUser.js:55]: this borrows work from https://github.com/flexion/ef-cms/pull/6870

File: web-client/switch-colors.sh [hunk 5]
```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
 
+[ -z "${ENV}" ] && echo "You must have ENV set in your environment" && exit 1
```
Comment (mmarcotte) [inline web-client/switch-colors.sh:3]: for when this is run locally, it's helpful to short circuit if the environment isn't properly configured

File: .circleci/config.yml (hunk 1 above)
Comment (adunkman) [reply .circleci/config.yml:954]: Partial revert of https://github.com/ustaxcourt/ef-cms/pull/482, linking in for future reference.

File: shared/src/persistence/dynamo/users/createMigratedPetitionerUser.test.js [hunk 6]
```diff
@@ -46,6 +57,7 @@ describe('createMigratedPetitionerUser', () => {
     expect(
       applicationContext.getCognito().adminCreateUser,
     ).toHaveBeenCalledWith({
+      MessageAction: 'SUPPRESS',
```
Comment (adunkman) [inline shared/src/persistence/dynamo/users/createMigratedPetitionerUser.test.js:60]: 👏

**Review by adunkman — APPROVED** [review]
👍 Thanks for the test updates!

**Pull request merged**